sdk/OoyalaSkinSDK/widgets/VideoViewPlayPause.js

```diff
@@ -132,7 +132,7 @@ var VideoViewPlayPause = React.createClass({
   },
 
   _renderLoading: function() {
-    if((this.props.rate <= 0 || this.props.playhead == 0) && !this.props.showButton && !this.state.controlPlaying) {
+    if(this.props.playhead == 0 && !this.props.showButton && !this.state.controlPlaying) {
```

Comment:
this won't work, because it effectively won't show the loading spinner at all except for the beginning (when playhead is 0)

we'll need to think of a better way to ensure the loading spinner is only showing when we want it to